### DISCUSSION TITLE
Client Version

### DISCUSSION THREAD

**Discussion by Nutanator** [ORelio/Minecraft-Console-Client#2049] [Q&A] [answered]
Anyone know if/when mcc will support 1.19?. I've looked around for help on this but I couldn't find anything.

**Comment by ORelio** [comment, marked as answer]
We are lacking maintainers / contributors for implementing new MC versions so no ETA is available.
Status can be found here: #1629

**Reply by Nutanator** [reply]
Ok thanks :D

**Reply by seepetey** [reply]
> 

I don't know how to code or else I would definitely help out.

 Would it help to hire someone on Upwork to do this?

**Reply by ORelio** [reply]
We have zero budget, it's a free open source project.
